projects/core/src/checkout/store/actions/replenishment-order.action.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ReplenishmentOrder,
+  ScheduleReplenishmentForm,
+} from '../../../model/replenishment-order.model';
+import { PROCESS_FEATURE } from '../../../process/store/process-state';
+import { StateUtils } from '../../../state/utils/index';
+import { SCHEDULE_REPLENISHMENT_ORDER_PROCESS_ID } from '../checkout-state';
+
+export const SCHEDULE_REPLENISHMENT_ORDER =
+  '[Checkout] Schedule Replenishment Order';
+export const SCHEDULE_REPLENISHMENT_ORDER_SUCCESS =
+  '[Checkout] Schedule Replenishment Order Success';
+export const SCHEDULE_REPLENISHMENT_ORDER_FAIL =
+  '[Checkout] Schedule Replenishment Order Fail';
+export const CLEAR_SCHEDULE_REPLENISHMENT_ORDER =
+  '[Checkout] Clear Schedule Replenishment Data';
+
+export class ScheduleReplenishmentOrder extends StateUtils.EntityLoadAction {
```

Comment:
Why is this an entity?

Reply:
To use the process state when clicking 'schedule a replenishment order'

Reply:
I thought of something, it's ok